### DISCUSSION TITLE
v0.0.14 - is there any male pre-trained model avaible for that version?

### DISCUSSION THREAD

**Discussion by 4lekz** [coqui-ai/TTS#539] [General Q&A] [answered]
I tried to use models listed by list_model I get
Use 🐸TTS <= v0.0.13 for this model. Current version does not support phoneme based models.
So what are avaible voices for v0.0.14 that I could use?

**Comment by erogol** [comment, marked as answer]
check `.model.json`. Any model set `use_phonemizer: False` works with v0.0.14.
